lib/rust/ensogl/core/src/display/world.rs

```diff
@@ -194,6 +202,30 @@ fn gather_shaders() -> HashMap<&'static str, shader::Code> {
 
 
 
+// ===================================
+// === Eager shaders instantiation ===
+// ===================================
+
+/// Begin compilation of all shaders that will be needed for the default scene.
+#[profile(Task)]
+pub fn instantiate_shaders() {
+    SHAPES_DEFINITIONS.with_borrow(|shapes| {
+        for shape in shapes {
+            let path = &shape.definition_path;
+            let preload = match path {
+                _ if path.starts_with("app/gui/view/debug_scene/") => false,
+                _ if path.starts_with("lib/rust/ensogl/example/") => false,
+                _ => true,
+            };
+            if preload {
+                let _shape = (shape.cons)();
+            }
+        }
+    });
+}
```

Comment:
This looks super hacky to me. What about this:
1. Renaming `app/gui/view/debug_scene/` to `app/gui/view/examples/`
2. Renaming `lib/rust/ensogl/example/` to `lib/rust/ensogl/examples/`
3. Excluding here anything that has folder `examples` in path.
4. Writing that in our docs?

Reply:
That's a bit less hacky! Implemented.